private-templates-service/client/src/components/SideBar/SideBar.tsx

```diff
@@ -98,7 +98,7 @@ const navMenuLinks: INavLinkGroup[] = [
       },
       {
         name: "New Template",
-        url: "/designer",
+        url: "/designer/newcard/1.0",
```

Comment:
can we extract this into `const newTemplateURL = "/designer/newcard/1.0"` and reference that here and on line 158 in this file? Just so that we don't break anything by accident if this URL ever changes.